shoop/core/migrations/0020_services_and_methods.py

```diff
@@ -0,0 +1,409 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import filer.fields.image
+import jsonfield.fields
+import shoop.core.fields
+from shoop.core.models import ShopStatus
+
+
+def create_default_service_providers(apps, schema_editor):
+    """
+    Create default carrier and payment processor.
+
+    Link existing ShippingMethods and PaymentMethods which use the
+    previous default module to the created carrier or payment processor.
+
+    Update polymorphic_ctypes for CustomerCarrier and
```

Comment:
This method does not update polymorphic_ctypes anymore. Can you move the this comment.